health-checker/src/main/java/io/stargate/health/DataStoreHealthChecker.java

```diff
@@ -20,55 +20,75 @@
 import io.stargate.db.datastore.DataStoreFactory;
 import io.stargate.db.datastore.ResultSet;
 import io.stargate.db.datastore.Row;
-import java.util.UUID;
+import io.stargate.db.query.builder.Replication;
+import io.stargate.db.schema.Column;
+import io.stargate.db.schema.ImmutableColumn;
+import io.stargate.db.schema.ImmutableTable;
+import io.stargate.db.schema.Table;
+import java.util.concurrent.ExecutionException;
 import java.util.concurrent.Future;
 import org.osgi.framework.BundleContext;
 import org.osgi.framework.ServiceReference;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class DataStoreHealthChecker extends HealthCheck {
-  private static final Logger logger = LoggerFactory.getLogger(DataStoreHealthChecker.class);
-
+  private static final Logger logger = LoggerFactory.getLogger(StargateNodesHealthChecker.class);
+  private static final String KEYSPACE_NAME = "data_store_health_check";
+  private static final String TABLE_NAME = "dummy_table";
+  public static final String PK_COLUMN_VALUE = "dummy_pk";
+  public static final String VALUE_COLUMN_VALUE = "dummy_value";
```

Comment:
These all need to be configurable.

Reply:
keyspace and table name? Sure, I can load it from `System.property` with those defaults. However, I think that `PK_COLUMN_VALUE` and `VALUE_COLUMN_VALUE` don't need to be configured.